### PR TITLE
Add unique request id

### PR DESCRIPTION
**Pull request by yiiliveext**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Tests pass?   | ✔️

### CODE REVIEW

**Comment by samdark** [comment]
There's https://github.com/yiisoft/yii-web/blob/master/src/Middleware/TagRequest.php for the purpose.

**Comment by yiiliveext** [comment]
> There's https://github.com/yiisoft/yii-web/blob/master/src/Middleware/TagRequest.php for the purpose.

It is optional. This PR is for another PR.

File: src/Middleware/TagRequest.php [hunk 1]
```diff
@@ -19,6 +19,6 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
 
     private function getRequestTag(): string
     {
-        return dechex(microtime(true) * 1000000);
+        return $request = uniqid('yii-request-', true);
```
Comment (samdark) [inline src/Middleware/TagRequest.php]:
```suggestion
        return uniqid('', true);
```

Comment (yiiliveext) [reply src/Middleware/TagRequest.php]: Done

**Pull request merged**